Exiled.CustomRoles/API/Features/CustomRole.cs

```diff
@@ -104,6 +109,21 @@ public abstract class CustomRole
         /// </summary>
         protected virtual bool KeepRoleOnDeath { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating the role when player escape as cuffed.
+        /// </summary>
+        protected virtual RoleType EscapeAsCuffed { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating the role when player escape hasn't cuffed.
+        /// </summary>
+        protected virtual RoleType EscapeAsUnCuffed { get; set; }
```

Comment:
Same as above, if we're adding the ability to have custom escape mechanics for custom roles, we should add the logic to process them within EXILED.